src/ascii_seq.rs

```diff
@@ -382,11 +382,10 @@ impl SeqVec for AsciiSeqVec {
     }
 
     fn random(n: usize) -> Self {
-        let mut rng = rand::rng();
+        let mut seq = vec![0; n];
+        rand::rngs::SmallRng::from_os_rng().fill_bytes(&mut seq);
```

Comment:
hmm; this generates 4x the randomness we need. We could instead generate 4x less and then only use 2 bits at a time in the loop below. But probably it's fine as it. 

If we really wanted to optimize this we might be able to replace the array indexing below with simd shuffles but not feeling like that now.

Reply:
Actually, what might be faster is to make an 256 long array mapping bytes to `[u8; 4]`, and then appending 4 values at a time. That's still relatively clean and should be fast.